coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -36,11 +36,13 @@ def _extract_doc_comment_simple(content, line, column, markers):
     while line < len(content):
         pos = content[line].find(markers[2])
         if pos == -1:
-            doc_comment += content[line][align_column:]
+            if content[line] == "\n":
+                doc_comment += "\n"
+            else:
+                doc_comment += content[line][align_column:]
```

Comment:
maybe we should check whether `conent[line][align_column:]` is empty? In this case even not right indented code can't fail (the examples have 4-spaces indentation, I'm highlighting them explicitly with dots):

```
....my documentation

..x

.it will start here
```

would then extract:
```
my documentation



will start here
```